### ISSUE TITLE
Not clear what the input TRAPI results URL should look like

### ISSUE THREAD

**Issue by ayushi-agrawal-gladstone** [wikipathways/BioThings_Explorer_PFOCR_prioritization#4]
It is not clear that a JSON TRAPI result URL is expected by the notebook as input. 
Suggested solutions:

1. In the user input widget, "TRAPI Result URL" field could be renamed to "TRAPI Result URL (JSON)"
2. At the top of notebook, provide more details that a JSON TRAPI results URL is needed as input. This is not clear from the current description. 

Reported by @khanspers

**Comment by AlexanderPico** [comment]
Good point. Or we could accept either, or even just the hash alone, and then construct the JSON URL?

**Comment by ayushi-agrawal-gladstone** [comment]
I like the idea of accepting either a hash or a JSON URL.
